src/apiCalls.js

```diff
@@ -1,7 +1,35 @@
-
+const handleError = (response) => {
+  if(!response.ok) {
+    throw `${response.status} ${response.statusText}`
+  } else {
+    return response.json()
+  }
+}
 
 export const getAllCourses = () => {
 
   return fetch('https://discify-api.herokuapp.com/api/v1/courses')
-    .then(response => response.json())
+    .then(response => handleError(response))
+}
+
+export const addNewPlayer = (name) => {
+
+  return fetch('https://discify-api.herokuapp.com/api/v1/players', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      player: {
+        name: name
```

Comment:
For instances like this where the property is the same and the variable holding the value, you can use object property value shorthand. `player: { name }`